### DISCUSSION TITLE
Hardcoding APBS multigrid data format output

### DISCUSSION THREAD

**Discussion by AlexanderKocev** [Electrostatics/apbs#274] [General]
Hello, I was wondering if it is possible to change the APBS code in such a way that the output OpenDX scalar multigrid data format can have preset values? For example, is it possible to preset the delta x, delta y, and delta z values to 0.5 Angstroms exactly, regardless of the  input protein structure? Conversely, is it possible to change the nx, ny and nz values to have a preset number of points, or the x, y and z coordinates of the origin? Are manipulations of this sort feasible or will they break APBS's calculations? And if possible, how would one implement them?

**Comment by sobolevnrm** [comment]
Hello --

Can you accomplish your goal by postprocessing the standard output with the [mergedx2](https://apbs.readthedocs.io/en/latest/using/tools.html?highlight=mergedx#mergedx-and-mergedx2) tool provided with APBS?

Thank you,

Nathan
